RxCocoa/macOS/NSButton+Rx.swift

```diff
@@ -20,25 +20,31 @@ extension Reactive where Base: NSButton {
         return controlEvent
     }
 
-    /// Reactive wrapper for `state` property`.
-    public var state: ControlProperty<Int> {
-        return NSButton.rx.value(
-            base,
-            getter: { control in
-                #if swift(>=4.0)
-                    return control.state.rawValue
-                #else
+    #if swift(>=4.0)
```

Comment:
@kzaher Thank you answer.
If support swift3.2, we will need this term.
I think it's good to keep it now, and if you decide don't support 3.2, please remove this term.